plugin/pkg/scheduler/util/utils.go

```diff
@@ -59,6 +61,12 @@ func GetUsedPorts(pods ...*v1.Pod) map[string]bool {
 	return ports
 }
 
+// PodPriorityEnabled indicates whether pod priority feature is enabled.
+func PodPriorityEnabled() bool {
+	// bsalamat: FIX THIS BEFORE MERGING!!!
+	return true || feature.DefaultFeatureGate.Enabled(features.PodPriority)
```

Comment:
I have intentionally force-enabled the feature to make sure that our tests pass with PriorityQueue. I will disable it before merging.